runtime/j9vm/CMakeLists.txt

```diff
@@ -56,6 +56,13 @@ target_link_libraries(jvm_common
 		${OMR_PLATFORM_THREAD_LIBRARY}
 )
 
+if(NOT JAVA_SPEC_VERSION LESS 24)
+	target_link_libraries(jvm_common
+		INTERFACE
+			j9thr
```

Comment:
Is it possible to move the implementation of `JVM_TakeVirtualThreadListToUnblock` to continuationHelpers and call it in javanextvmi.cpp via the internalVMfunctions table?